package.json

```diff
@@ -11,8 +11,9 @@
   },
   "license": "ISC",
   "dependencies": {
-    "@applitools/eyes-selenium": "^4.53.0",
-    "mocha": "^8.3.2",
+    "@applitools/eyes-selenium": "^4.55.2",
+    "chromedriver": "^90.0.1",
```

Comment:
same thing with this one as the other repo https://github.com/applitools/tutorial-selenium-javascript-basic/pull/11#discussion_r641730174